### ISSUE TITLE
When to update the next version？

### ISSUE THREAD

**Issue by xiesi88** [googlevr/cardboard#218]
Thank you very much for your work. We are eager to solve the following problems. It's a long time since the last version. We'd like to know when to update the next version.

![image](https://user-images.githubusercontent.com/27883184/109479575-7b3a0000-7ab5-11eb-9fff-59ae82541edf.png)

**Comment by UnayOzan** [comment]
Having the same problems on my released project, waiting for update. These are needs to be fixed ASAP.

Thanks for the dev team in advance.

**Comment by andyz** [comment]
Like is this even still being worked on?
It has been 3 months... are Google not interested in cardboard any more?

**Comment by jballoffet** [comment]
As explained in #205, we're blocking release v.1.5.0 until we have completed the Metal integration. We are currently working on it (we have already finished integrating it to iOS natively and we are now solving some issues in the Unity integration via the XR Plugin). Once we have finished the integration and we have kicked off the QA process, I'll reply on issue #11 to communicate this. I’d recommend you star or watch it to be notified. Thanks for reaching out!
